sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala

```diff
@@ -207,10 +207,19 @@ object OrcUtils extends Logging {
   }
 
   /**
-   * @return Returns the result schema string based on the canPruneCols flag.
-   *         resultSchemaString will be created using resultsSchema in case of
-   *         canPruneCols is true and for canPruneCols as false value
-   *         resultSchemaString will be created using the actual dataSchema.
+   * This method returns the result schema as string based on the canPruneCols flag.
+   * resultSchemaString will be created using pruned col in case of
+   * canPruneCols is true and for canPruneCols as false value
+   * resultSchemaString will be created using the actual dataSchema.
+   * Also as the part of this method update the MAPRED_INPUT_SCHEMA
```

Comment:
`update the MAPRED_INPUT_SCHEMA` -> `updates the MAPRED_INPUT_SCHEMA in the given 'conf'`.

Reply:
Updated the comment as suggested by @HyukjinKwon